### PR TITLE
Mahesh Profile Added

### PR DESCRIPTION
**Pull request by Mahesh-Kasabe**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2442"></a>

### CODE REVIEW

**Review by accodes21 — APPROVED** [review]
Looks great

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

File: data/mahesh.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**Open Source** Advocate | DevOps ",
+  "tags": ["Open Source", "DevOps", "Kubernetes", "Docker", "Node.js"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/maheshstwt" },
+    { "platform": "github", "url": "https://github.com/Mahesh-Kasabe" },
+    { "platform": "website", "url": "https://mahesh-kasabe.github.io/" }
+  ],
+  "avatar": "https://avatars.githubusercontent.com/u/60398112?s=400&u=f0abce668aeb1611b03f279d9596c5674ab757c6&v=4",
```
Comment (Panquesito7) [inline data/mahesh.json:12]: You can use this link instead to have it shorter and updated automatically. 🙂
```suggestion
  "avatar": "https://github.com/Mahesh-Kasabe.png",
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**